ratpack-kafka-producer/src/main/java/smartthings/ratpack/kafka/KafkaProducerModule.java

```diff
@@ -73,5 +80,53 @@ public boolean isEnabled() {
 		public void setEnabled(boolean enabled) {
 			this.enabled = enabled;
 		}
+
+		public void setLingers_ms(Optional<Long> lingers_ms) {
```

Comment:
Does optional work in the setters assuming this is being bound in the config?  This works for me if so, but wanted to verify.

It might be good to add a simple test that setting these values builds up a properties object the way you expect.  Since right now that is not the case.